webapp/src/App.tsx

```diff
@@ -10,6 +10,7 @@ function App() {
         <p>
           Edit <code>src/App.tsx</code> and save to reload.
         </p>
+        <p>{`Hello, ${process.env.REACT_APP_NAME}!`}</p>
```

Comment:
So, as far as I understand, this PR is still not connecting frontend to backend, is it? REACT_APP_NAME is not anything we pull for the backend API, just an environment variable that we set in the webapp container. There are not calls to the backend API.
Or did I miss something?

Reply:
Correct, the webapp frontend is at the `/` location, and the transcendence-app backend is at the `/api` location, and it has one route `/api/v1/hello`.

`REACT_APP_NAME` and `TRANSCENDENCE_APP_NAME` are placeholders used to test that Docker passes the environment variables to the containers.